runtime/ftplugin/yaml.vim

```diff
@@ -17,19 +17,20 @@ setlocal comments=:# commentstring=#\ %s expandtab
 setlocal formatoptions-=t formatoptions+=croql
 
 " rime input method engine uses `*.custom.yaml` as its config files
+if !exists("g:yaml_recommended_style") || g:yaml_recommended_style != 0
```

Comment:
```suggestion
if get(g:, "yaml_recommended_style",1)
```

But is seems `yaml_recommended style` is not documented.

Reply:
> But is seems `yaml_recommended style` is not documented.

True. I guess this is some kind of switch, not some kind of code style?

https://github.com/vim/vim/commit/b17893aa940dc7d45421f875f5d90855880aad27#diff-c18dc485af790bb2f32faca58fbc83e462501d3229b3335e18c20ed4350e6333